### ISSUE TITLE
Unknown Error 

### ISSUE THREAD

**Issue by jatinkumar31** [r0ld3x/terabox-app#17]
Unknown Error

coming

even when i use premium account cookies

**Comment by r0ld3x** [comment]
yeah terabox is updates their security. Finding a way. till then stay tuned

**Comment by jatinkumar31** [comment]
Please fix it

**Comment by r0ld3x** [comment]
I am trying my best @jatinkumar31 :)

**Comment by Cyberwalaguy** [comment]
can you provide me the api key I have vps with 1. year plan

**Comment by r0ld3x** [comment]
> can you provide me the api key I have vps with 1. year plan

terabox api has been upgraded. I am finding solutions

**Comment by mugnimaestra** [comment]
> > can you provide me the api key I have vps with 1. year plan
> 
> terabox api has been upgraded. I am finding solutions

have you found the solution yet?

**Comment by r0ld3x** [comment]
> > > can you provide me the api key I have vps with 1. year plan
> > 
> > 
> > terabox api has been upgraded. I am finding solutions
> 
> have you found the solution yet?

oh yes
use premium account's cookie
